docs/stdlib/select.md

```diff
@@ -66,3 +66,35 @@ Options      |         Description           | Required?
 
 Only the point with the exact median value for the `field` name
 provided will be forwarded.
+
+## top - sub
+
+```
+... | select.top -n number -by field | ...
+```
+
+Options  |                              Description                              | Required?
+-------- | --------------------------------------------------------------------- | --------- :
+`n`      | how many of the fields with the highest values of `by` to keep        | Yes
+`by`     | name of the field to sort by before figuring out the top `n` to keep  | Yes
+
+Let the `n` points that have the highest value of `by` pass through. 
```

Comment:
This should mention sorting. As it is, the text implies not touching the points at all beside filtering.